subprojects/plugins/src/main/groovy/org/gradle/api/tasks/testing/Test.java

```diff
@@ -745,13 +745,20 @@ public Test exclude(Closure excludeSpec) {
      *
      * For more information on supported patterns see {@link TestFilter}
      */
-    @Option(option = "tests", description = "Sets test class or method name to be included, '*' is supported.")
+    @Option(option = "tests-include", description = "Sets test class or method name to be included, '*' is supported.")
```

Comment:
Interesting, so `Option` is how we can reconfigure some gradle task attributes via command line?